xla/tests/hlo_test_base.h

```diff
@@ -175,6 +175,18 @@ class HloTestBase : public ManifestCheckingTest {
       std::optional<absl::string_view> expected,
       std::function<void(HloModule*)> after_pass_checks = nullptr);
 
+  void RunAndFilecheckHloRewriteWithConfig(
```

Comment:
Instead of adding two new methods, add a `const HloModuleConfig* = nullptr` optional argument to `RunAndFilecheckHloRewrite`.

Reply:
Resolved.